GoogleForCreators/web-stories-wp
# Disable Update button if there are no changes to be saved

## Context

Before this PR the update button would stay enabled even though there werent new changes. It was because the update button was meant to be disabled only if there was any `uploading` or any `saving` action happening.

## Summary

This PR changes the condition and adds a unit test to check the update button works properly.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Open a published story or any draft story in story-editor
2. Dont make any changes you should see that the Update button is disabled.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #3548
